packages/python/patch-no-srand

```diff
@@ -0,0 +1,23 @@
+--- Modules/expat/xmlparse.c
++++ Modules/expat/xmlparse.c
+@@ -20,7 +20,9 @@
+ #include <string.h>                     /* memset(), memcpy() */
+ #include <assert.h>
+ #include <limits.h>                     /* UINT_MAX */
++#if !defined(__CloudABI__)
```

Comment:
I'm not sure I understand. Why include `time.h` when I've `#ifdef` disable the only use of it?